code/clue.js

```diff
@@ -1,59 +1,158 @@
+}
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'She always watches instagram',
+  age: 25,
+  image: 'assets/scarlet.png',
+  occupation: 'Influencer' 
 }
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'grey',
+  description: 'he is always late',
+  age: 48,
+  image: 'assets/mustard.png',
+  occupation: 'lawyer' 
+}
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'lila',
+  description: 'She always picks up packages',
+  age: 55,
+  image: 'assets/peacock.png',
+  occupation: 'Doctor' 
+}
+const mrsWhite = {
+  firstName: 'Ana',
+  lastName: 'White',
+  color: 'Orange',
+  description: 'she is always one step behind you',
+  age: 35,
+  image: 'assets/white.png',
+  occupation: 'garderner' 
+};
+
+const weapons = [
+'rope',
+'knife',
+'candlestick', 
+'dumbbell', 
+'poison', 
+'axe', 
+'bat',
+'throphy', 
+'pistol' 
+];
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const rooms = [
+'diningRoom', 
+'conservatory', 
+'kitchen', 
+'study', 
+'library', 
+'billiardRoom', 
+'lounge', 
+'ballroom', 
+'hall', 
+'spa', 
+'livingRoom', 
+'observatory', 
+'theater', 
+'guestHouse', 
+'patio'];
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+// Group OBJECTS (just 'suspects' because i choosed to no give the weapons a discirption (not at least in this point)
 
 const suspects = [
   mrGreen,
+  professorPlum,
+  missScarlet,
+  colonelMustard,
+  mrsPeacock,
   mrsWhite
-  // ...  and the rest
-]
+];
 
-const weapons = []
-
-const rooms = []
-
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
+// FUNCTION RANDOMLY SELECT ONE ITEM FROM THE ARRAY 
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
 
-// This function will be invoked when you click on the killer card.
-const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
-  mystery.killer = randomSelector(suspects)
 
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
+const mystery = {
+  suspects,
+  weapons:[],
+  rooms:[] 
+};
+
+
+// FUNCTIONS + INVOKE FUNCTIONS
+
+// FUNCTIONS pickKiller
+const pickKiller = () => {
+  mystery.killer = randomSelector(suspects);killerImage.src = mystery.killer.image;
   document.getElementById('killerCard').style.background = mystery.killer.color
+  document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+};
+document.getElementById(
```

Comment:
I think it would be better if these lines of code 118-124 had the same style - either on one line like rows 118-120 or with breaks like the rows  122-124. I guess it happened because of the editor. I had the same issue and planning to change it:)

Reply:
I agree. Maybe this is caused by an auto-formatter?